assets/content/blog/2016-09-19 opening-doors.md

```diff
@@ -0,0 +1,18 @@
+# Opening the doors to the future
+
+> Hello everyone, we are excited to announce some important things from part of the HaxeDevelop team:
+
+As you can see we now have a new blog, thanks completely to [Mark Knol](https://twitter.com/mknol), props to him. 
+We expect this blog to serve as a successful way to notify all our new meaningful progress, experimental new features or possible ideas and at the same time allow all of you to raise your voice and help shaping HaxeDevelop.
+
+We have also released HaxeDevelop 5.2. This release fixes and improves several things and adds a few of new features. This is the [list of changes](http://haxedevelop.org/haxedevelop-5.2.html).
+
+* **[Click here to download it](http://haxedevelop.org/download.html).**
+
+As you can see this new version is considered to be minor but we believe it is an important release. We plan to use it as a point of change, even if there may be new minor releases fixing possible problems in HaxeDevelop&nbsp;5.2, work for HaxeDevelop&nbsp;6 _will start now_. 
+We will make bigger changes to parts of code, the application will move to .NET&nbsp;4.0 or possibly a newer version, and we will start accepting commits making use of C#&nbsp;6 syntax. 
```

Comment:
Could you change this line? Maybe:

We will make bigger changes to some parts of code, we will start accepting commits making use of C#&nbsp;6 syntax and changes possibly breaking backwards compatibility.

The move to .NET4+ will surely happen, but let's give it less importance for now.